dbcon/mysql/ha_mcs_execplan.cpp

```diff
@@ -6570,7 +6607,8 @@ bool isUpdateHasForeignTable(THD* thd)
 {
     LEX* lex = thd->lex;
 
-    if (lex->sql_command != SQLCOM_UPDATE_MULTI)
+    if ((lex->sql_command != SQLCOM_UPDATE_MULTI) &&
```

Comment:
What if we use isUpdateStatement() here?

Reply:
Done.